src/grants/use-cases/submit-application.use-case.js

```diff
@@ -66,6 +66,10 @@ const getAnswersInSchema = (clientRef, schema, answers) => {
 };
 
 export const submitApplicationUseCase = async (code, { metadata, answers }) => {
+  logger.debug(
+    `Application submitted for code ${code} and metadata ${JSON.stringify(metadata)} and answers ${JSON.stringify(answers)}`,
```

Comment:
This could leak PII if redaction doesn't catch it

Reply:
removed the metadata and answers and code should be suffice